src/common/utils/constants.ts

```diff
@@ -21,3 +22,234 @@ export const DEFAULT_WINDOW_WIDTH = 1000;
 export const DEFAULT_WINDOW_HEIGHT = 700;
 export const MINIMUM_WINDOW_WIDTH = 700;
 export const MINIMUM_WINDOW_HEIGHT = 240;
+
+export const localeTranslations: Record<string, string> = {
```

Comment:
Is it necessary to specify this as a `Record<string, string>` or can that be omitted? 0/5

Reply:
TS defaults it to `any` for some reason if I don't specify this.